vignettes/decorate-module-output.Rmd

```diff
@@ -210,12 +214,11 @@ failing_decorator <- teal_transform_module(
 
 ### Example Module
 
-To include decorators in a `teal` module, pass them as arguments (`ui_args` and `server_args`) to the module’s `ui` and 
-`server` components, where they will be used by `ui/srv_teal_transform_module`. 
+To include decorators in a `teal` module, pass them as arguments (`ui_args` and `server_args`) to the module’s `ui` and
+`server` components, where they will be used by `ui_transform_teal_data` and `srv_transform_teal_data`.
```

Comment:
@gogonzo am I using the right function name?

Reply:
Yes. Closing a conversation